test/validators/format.js

```diff
@@ -47,13 +47,34 @@ describe("format validation", () => {
       })
 
 
+      it("allows RegExp flag pattern", function() {
+
+        //zipcode regex
+        var pattern = new RegExp('^[0-9]{8}$')
```

Comment:
where is exactly the flag?

Reply:
sorry, my mistake, i forgot to update...